### DISCUSSION TITLE
Issues with Webconfig, DualShock 3, Wii CC Pro (Macros), and DualSense (PS5).

### DISCUSSION THREAD

**Discussion by Tye2K** [darthcloud/BlueRetro#514] [Q&A]
Hi. Got two Blue Retro Internal Adpaters for my GameCubes, installed one several days ago.

1. The WebConfig (blueretro.io) does not & will not find/connect to BlueRetro. It just gives me “Argh! Couldn’t find BlueRetro”. I have tried multiple times, and even held the reset button to make SURE it was in pairing mode, only to no success.
2. It says PlayStation 3 controllers (DualShock3/SixAxis) are compatible but they aren’t; it will not connect wirelessly and Googling did not help.
3. Macros (shortcuts) do not work on Wii Classic Controller Pro. I have no other controllers other than this & my brand new DualSense (shortcuts work on this controller) to test this issue with.
4. Speaking of DualSense, the triggers (L2/R2) do not respond/register correctly. The only way I can explain it is I’m able to air dodge (and quit matches with L+R+A+START) in Melee with my CC Pro just fine, but I _CANNOT_ do either of these things with my DualSense which is very annoying especially if I want to quit a match (I usually play 1P). It should be noted that shielding does still work, so the triggers **DO** work but not at its full capacity I guess. I don’t know if remapping/configuring controls would solve this issue but again, my PC will not connect to the unit so this is what I’m stuck with.

Hoping some attention gets brought to this because otherwise this is a really cool mod especially for a 2001 home console but after all the waiting, anticipation & excitement… this has so far put a really bad taste in my mouth.

**Comment by darthcloud** [comment]
1. Try to be closer or try different laptop or phone, close chrome tab/open new one between try.
2. Did you follow the pairing instructions? https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide#ps3
3. Are you pressing ZR+ZL+"+"+B for shutdown as example.
4. Did you program the merged trigger presets for having both analog and digital trigger map to the PS analog only trigger?
 https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#33---example-preset-load

More details why you need that preset here:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-System-Specific-User-Manual#mapping-15

Macros may interfere with some games, I'll make them configurable in a future release.

**Reply by Tye2K** [reply]
1. My PC sits underneath my GameCube (see below)
![DDD3DBBF-4B7A-42D1-8312-2F1A175EA782](https://user-images.githubusercontent.com/99521193/209363720-a62840a8-cb0e-493d-bf39-f8116aeae9d1.jpeg)
I also tried different tabs, restarted Chrome and even my PC.
2. No. As stated, I literally cannot use webconfig for anything.
3. No, before trying that I realized ZL/ZR are mapped to Z on the Wii CC. That being said, I tried using ZL+ZR as well as L+R just to be sure & still nothing.
4. Same as answer 2; I cannot do anything in regards to the web config. All of the problems I have stated have been experienced upon using BlueRetro after installation, before configuring anything (I chose to go this route just to be sure that if I did experience any problems, it wouldn’t have been on my part mostly).

**Reply by darthcloud** [reply]
The Bluetooth antenna will broadcast best in the forward direction were the controller port are, so your PC could be in a blind spot, try moving thing around or try with a phone.
